package/kernel/linux/modules/leds.mk

```diff
@@ -16,6 +36,26 @@ define KernelPackage/leds-gpio
   AUTOLOAD:=$(call AutoLoad,60,leds-gpio,1)
 endef
 
+
+define KernelPackage/led-group-multicolor
+	 SUBMENU:=$(LEDS_MENU)
+	 TITLE:=LEDs group multi-color support
+	 KCONFIG:= \
+	     CONFIG_LEDS_CLASS_MULTICOLOR \
```

Comment:
No, you have a separate kmod for this, just include it as a dependency.

Reply:
Ok I have removed that, and it test builds fine.